CheckDrive.Web/CheckDrive.Web/ViewModels/DashboardViewModel.cs

```diff
@@ -25,4 +151,43 @@ public class SpliteChartData
     public decimal Ai95 { get; set; }
     public decimal Diesel { get; set; }
 }
+public class Debt
+{
+    public int Id { get; set; }
+    public string Driver { get; set; }
+
+    [Newtonsoft.Json.JsonConverter(typeof(Newtonsoft.Json.Converters.StringEnumConverter))]
+    public PetrolType Petrol {  get; set; } 
+    public decimal FuelAmount{ get; set; }
+    public decimal PaidAmount{ get; set; }
+    public DebtStatus Status { get; set; }
+
+}
 
+public class PetrolCount
+{
+    [Newtonsoft.Json.JsonConverter(typeof(Newtonsoft.Json.Converters.StringEnumConverter))]
+    public PetrolType Type { get; set; }
+    public int Count { get; set; }
+}
+
+public enum DebtStatus
+{
+    Paid,
+    Unpaid,
+    PartiallyPaid
+}
```

Comment:
Can you replace this enum to models folder

Reply:
fixed